JavaScriptSDK.Enums/LoggingEnums.ts

```diff
@@ -0,0 +1,90 @@
+export enum LoggingSeverity {
+    /**
+     * Error will be sent as internal telemetry
+     */
+    CRITICAL = 1,
+
+    /**
+     * Error will NOT be sent as internal telemetry, and will only be shown in browser console
+     */
+    WARNING = 2,
+}
+
+/**
+ * Internal message ID. Please create a new one for every conceptually different message. Please keep alphabetically ordered
+ */
+export enum _InternalMessageId {
```

Comment:
I think we need to convert these to just plain object for the size issue that we had discussed.